### PR TITLE
[V3 Mod] User Facing Responses

### PR DESCRIPTION
**Pull request by mikeshardmind**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
This is a PR addressing currently respecting `embed_requested` and the positional/keyword formats as discussed in #1436 and #1526

### CODE REVIEW

**Comment by mikeshardmind** [comment]
Not ready for b14

**Comment by mikeshardmind** [comment]
I modified the scope of this PR to just the mod cog, I'm also opening an issue to track which cogs will still need similar PRs

**Review by Tobotimus — DISMISSED** [review]

File: redbot/cogs/mod/mod.py [hunk 1]
```diff
@@ -503,23 +514,25 @@ def __unload(self):
...
-                        "You have been temporarily banned from {} until {}. "
-                        "Here is an invite for when your ban expires: {}"
-                    ).format(guild.name, unban_time.strftime("%m-%d-%Y %H:%M:%S"), invite)
-                )
+                msg = _(
+                    "You have been temporarily banned from {g} until {t}. "
+                    "Here is an invite for when your ban expires: {i}"
+                ).format(g=guild.name, t=unban_time.strftime("%m-%d-%Y %H:%M:%S"), i=invite)
+                if await self.bot.embed_requested(user.dm_channel, user):
+                    await user.send(embed=discord.Embed(msg))
```
Comment (palmtree5) [inline redbot/cogs/mod/mod.py]: Won't work

**Review by palmtree5 — CHANGES_REQUESTED** [review]
Not that I particularly support embedding everything the bot sends, but if this is to be merged, the one comment here needs to be fixed because it will cause errors

**Comment by mikeshardmind** [comment]
This shouldn't be merged until the discussion about embeds in #advanced-coding discord channel is resolved and any needed changes after it are made.

**Pull request closed without merging**